engine/admin/ansible.md

```diff
@@ -0,0 +1,72 @@
+---
+aliases:
```

Comment:
Also, you need to update the `/_data/toc.yaml` file to incude a reference to the file you added, otherwise it won't be displayed in the navigation menu.